react/src/Sidebar/SidebarContext.tsx

```diff
@@ -0,0 +1,24 @@
+import { createContext, PropsWithChildren, useContext } from 'react'
+
+export interface SidebarContextProps {
+  activeId?: string
+}
+
+const SidebarContext = createContext<SidebarContextProps | undefined>(undefined)
+
+export const SidebarProvider = ({
+  children,
+  ...props
+}: PropsWithChildren<SidebarContextProps>) => {
+  return (
+    <SidebarContext.Provider value={props}>{children}</SidebarContext.Provider>
+  )
+}
+
+export const useSidebar = () => {
+  const context = useContext(SidebarContext)
+  if (!context) {
+    throw new Error('useDatePicker must be used within a DatePickerProvider')
```

Comment:
nit: `useSidebar/SidebarProvider`

Reply:
yup, not used anyways, will be used in future PRs. will remove for now